MyCryptoHQ/MyCrypto
# Update ENS Addresses (LEGACY)

Closes CH4827

## Description

Updates ENS stuff as dictated by https://docs.ens.domains/ens-migration/guide-for-dapp-developers

## Changes

* Removes ENS from navigation
* Updates ENS registry contract address across mainnet and testnet
* Doesnt do anything for RSK?

## Steps to Test

1. If you try to resolve migrated.eth and it works then youve done it right
2. Attempt to send a transaction on eth
3. Attempt to resolve an address on contracts page
4. Attempt to send a transaction on testnet
5. Make sure desktop builds dont have ENS in sidebar navigation
6. Make sure mobile doesn't have ENS navigation item

## Currently on mycrypto.com....
<img width="1282" alt=" 2020-02-03 at 1 05 44 AM" src="https://user-images.githubusercontent.com/7924827/73640527-40039e00-4623-11ea-8252-338be726d92a.png">

## With this PR:

<img width="1282" alt=" 2020-02-03 at 1 06 00 AM" src="https://user-images.githubusercontent.com/7924827/73640528-40039e00-4623-11ea-92dd-e710914d700b.png">
